src/commons/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Button(props) {
+  const { onClick, children } = props;
```

Comment:
이렇게만 처리해도 괜찮을 것 같아요~

```suggestion
function Button({ onClick, children }) {
```